controllers/user.py

```diff
@@ -40,6 +40,69 @@ def login_token():
     auth_jwt.verify_expiration = False
     return auth_jwt.jwt_token_manager()
 
+# ------------------------------------------------------------------------------
+def fill_details():
+    """
+        Fill the details after google registration 
+    """
+    auth_token = request.vars.get('g_token')
+    email = request.vars.get('email')
+
+    #verify parameters
+    if not email or not auth_token:
+        raise HTTP(400, 'Invalid parameters')
+        return
+    #verify credentials
+    user_info = json.loads(current.REDIS_CLIENT.get("g_token_"+email))
```

Comment:
what if the token is not found? (Please handle it)

Reply:
if the token is not found it will be null
and according to if the condition on line 57 it will not be equal to not nullable field auth_token and thus an error will be thrown back